catalog/ui/catalog-ui-search/src/main/webapp/styles/components/layers.less

```diff
@@ -14,8 +14,10 @@
         width: 100%;
         margin-bottom: 0px;
     }
+
     input[type="checkbox"] {
-        display: none;
+        vertical-align: middle;
+        padding: 0px;
```

Comment:
This should be `margin: 0px` instead of padding.  That'll likely fix your cross browser issues.  If it doesn't, let me know.